BEST_PRACTICES.md

```diff
@@ -203,3 +216,459 @@ Use a simple NSURLSession wrapper to make things simpler, [Networking](https://g
 Views should be layout using Apple's Auto Layout. No third-party frameworks are recommeded at the moment but this is open for change. Just try to use the highest abstraction that's available to you, wheter this is `UIStackView` or  `NSLayoutAnchor`.
 
 Old style layout is still an option for highly dynamic layouts such as when subclassing UICollectionViewLayout, but in general lines, first try Auto Layout.
+
+# Swift Style Guide
+
+Our overarching goals are conciseness, readability, and simplicity.
+
+## Naming
+
+Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names, variables and constants should start with a lower case letter.
```

Comment:
can we add a clause to avoid at all costs doing objc-style namespacing too? `project_functionName()`